python/ray/workflow/http_event_provider.py

```diff
@@ -0,0 +1,263 @@
+import asyncio
+from typing import Dict
+from fastapi import FastAPI, Request
+from fastapi.responses import JSONResponse
```

Comment:
What I mean is to move this to HTTPEventProvider

Reply:
@iycheng We investigated the error from the earlier build was incurred in workflow_access.py. With lazy import moved inside `create_http_event_provider()`, the build error due to fastapi dependency went away. (fastapi is in requirements.txt)